homeassistant/components/unifi/update.py

```diff
@@ -23,11 +28,88 @@
 from .const import ATTR_MANUFACTURER, DOMAIN as UNIFI_DOMAIN
 
 if TYPE_CHECKING:
+    from aiounifi.models.event import EventKey
+
     from .controller import UniFiController
 
+Data = TypeVar("Data")
+Handler = TypeVar("Handler")
```

Comment:
Can these be private? And could you adjust naming to end with `T`?
```suggestion
_DataT = TypeVar("_DataT")
_HandlerT = TypeVar("_HandlerT")
```

Reply:
For this PR it can be internal. It will change once I consolidate the implementation between platforms. So.maybe unnecessary?